src/pages/styles.module.css

```diff
@@ -32,13 +40,33 @@
 }
 
 .featureImage {
-  height: 130px;
-  width: 130px;
+  height: 90px;
+  width: 90px;
 }
 
 .featuresGrid {
   display: grid;
   grid-template-columns: repeat(auto-fit, minmax(360px, 1fr));
-  grid-auto-rows: 330px;
-  gap: 4px;
+  grid-auto-rows: 250px;
+  gap: 2.5rem;
+}
+
+/* new styles */
+
+.padmd {
+  padding: 0 2rem;
+}
+
+.featuresp {
+  line-height: 1.4;
+}
+.featuresh3 {
+  line-height: 1.4;
+  font-size: 24px;
+}
+
+.rowflex {
+  gap: 30px;
+  display: flex;
+  justify-content: center;
```

Comment:
Similar to my previous comment about utility vs semantic, this utility class is only used once. We should instead apply the style through a semantically named class (e.g. "buttonWrapper" or something).